backend/src/main/java/ca/bc/gov/hlth/mohums/controller/MetricsController.java

```diff
@@ -58,6 +59,36 @@ public List<Map<String, Object>> getActiveUserCount() throws SQLException {
         return jdbcTemplate.queryForList(sql);
     }
 
+    @GetMapping("/metrics/total-active-user-count/{format}")
+    public List<Map<String, Object>> getTotalActiveUserCountWeek(@PathVariable String format) throws SQLException {
```

Comment:
We have integration tests at `moh-keycloak-user-management\backend\src\test\java\ca\bc\gov\hlth\mohums\integration\MoHUmsIntegrationTests.java`. Please add at least a smoke test for this SQL query.

As it stands, a simple typo in the query would go unnoticed until someone manually tests it. A "smoke test" is a low cost way to identity trivial but irritating and potentially costly mistakes.

https://en.wikipedia.org/wiki/Smoke_testing_(software)